packages/client/tests/memory/query/test.ts

```diff
@@ -0,0 +1,11 @@
+import { createMemoryTest } from '../_utils/createMemoryTest'
+
+//@ts-ignore
+type PrismaModule = typeof import('./.generated/node_modules/@prisma/client')
+
+void createMemoryTest<PrismaModule>(async ({ PrismaClient }) => {
+  const client = new PrismaClient()
+  await client.$connect()
+  await client.user.findMany({})
```

Comment:
Do you have an idea in mind of all the possible things we should be testing? If we don't have time to write all these, we could open tasks in order to achieve it over time.

Reply:
Generally, I think it might make sense to test the scenarios where we now client is doing something really differently from existing cases. For example, I don't think it makes sense to test different queries, as they more or less boil down to the same request to the engine. These 3 are first scenarios that came to mind, not sure what we can add right now, but if anybody has any ideas I'd be happy to implement them now — writing these types of tests does not take that much time.